mobile/src/src/components/Community/Feed/EventView.jsx

```diff
@@ -19,8 +20,12 @@ const EventView = ({
   photo,
   is_interested,
   is_attending,
+  attendEvent,
+  interestedEvent,
 }) => {
   const { width, height } = Dimensions.get("window");
+  const [attending, setAttending] = useState(is_attending);
+  const [interested, setInterested] = useState(is_interested);
```

Comment:
If we set and update the 'is_attending' and 'is_interested' props from 1 level up, then maybe we don't need to manage state inside this functional component, since react always rerenders components when props change.

Reply:
True true. Ill look into this